client/fe/webpack/moduleRules.js

```diff
@@ -160,13 +160,16 @@ const moduleRules =
         {
           loader: 'sass-loader',
           options: {
-            implementation: require('sass-embedded'),
-            additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
+            api: 'modern',
```

Comment:
Just wondering: Why not 'modern-compiler', as it is strongly recommended?

> Using modern-compiler and sass-embedded together significantly improve performance and decrease built time. We strongly recommend their use. We will enable them by default in a future major release.

Reply:
oh no. Something went wrong. These changes where not supposed to be here in this PR.

Reply:
When I use "modern-compiler" the build doesn't finish. No errors, no output, but it doesn't stop.

Reply:
added the workaround @spiess-demos mentioned here: 2179f0d7194c62e1c74e18fe509a829a52ce53f7